src/main/java/edu/kis/powp/jobs2d/factories/TriangleFactory.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.jobs2d.factories;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.command.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.command.SetPositionCommand;
+
+public class TriangleFactory {
+  public static DriverCommand triangle(Job2dDriver driver) {
```

Comment:
Squere and triangle factories can be easily refactored to Factory Method pattern